README.md

```diff
@@ -93,12 +93,16 @@ require('alexa-app-server').start({
 	// root of your web server.
 	app_root: "/alexa/",
 
-	// The directory containing server-side processing modules (see below)
+	// The directory containing server-side processing modules (see below).
 	server_dir: "server",
 
-	// The port the server should bind to
+	// The port the server should bind to.
 	port: 80,
 
+	// The host address in which the server should bind to. If not specified,
+	// this argument will be ignored.
+	host_address: undefined,
```

Comment:
For the purposes of this PR I agree not to change it, opened https://github.com/alexa-js/alexa-app-server/issues/38.